wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/FileLinker.kt

```diff
@@ -52,6 +52,7 @@ internal class FileLinker(
     for (path in paths) {
       if (sink.add(path)) {
         val fileLinker = linker.getFileLinker(path)
+        addImportsRecursive(sink, fileLinker.protoFile.imports)
```

Comment:
Without this, we would not recursively load "yellow" here. I don't think it would add any substantial work even when we are not exhaustively loading

Reply:
Not sure

Reply:
I think this is the place to check loadExhaustively

Reply:
Fixed!